### PR TITLE
Sheet CSS modules

### PR DESCRIPTION
**Pull request by bhollis**
Move Sheet CSS to modules, which also solves the specificity problems we've had with them.

### CODE REVIEW

**Review by robojumper — CHANGES_REQUESTED** [review]
Once again seems to break the mobile item popup sheet header 
grafik

**Comment by bhollis** [comment]
Unfortunate - my intention was actually to fix this sort of specificity problem by flattening the original sheet classes, and then relying on the fact that the sheet classes would appear before the sheet-modifier classes. Unfortunately the styles don't seem to be in import-graph order.

I wish I could use `@layer` but we can't as long as we're stuck with Chrome 85 from Steam.

**Comment by bhollis** [comment]
I "solved" this by importing the stylesheet directly in `Index.tsx` which makes it load higher up in the overall stylesheet. Someday we'll have `@layer` and can fix this the right way.

File: src/app/_variables.scss [hunk 1]
```diff
@@ -129,22 +129,15 @@ $emblem-height: 48px;
...
-  border-color: dim-rgb-values-to-rgba(var(--my-base-color-rgb), $alpha: 0.75);
-
-produces
-
-  --my-base-color-rgb: 255, 128, 0;
-  background-color: rgba(255, 128, 0, 1);
-  border-color: rgba(255, 128, 0, 0.75);
-
-*/
+// Utility functions to allow for augmenting a hex color value with an alpha component.
```
Comment (bhollis) [inline src/app/_variables.scss:132]: I snuck this change in here, but as I was debugging I noticed that this big block comment was getting output in every file, which seemed unnecessary.

**Review by robojumper — APPROVED** [review]

**Pull request merged**